app/controllers/CheckYourAnswersController.scala

```diff
@@ -17,30 +17,67 @@
 package controllers
 
 import com.google.inject.Inject
-import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import controllers.actions._
+import controllers.routes._
+import pages.{SubscriptionAmountAndAnyDeductions, TaxYearSelectionPage}
 import play.api.i18n.{I18nSupport, MessagesApi}
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import services.SubmissionService
+import uk.gov.hmrc.http.HttpResponse
 import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
 import utils.CheckYourAnswersHelper
 import viewmodels.AnswerSection
 import views.html.CheckYourAnswersView
 
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
 class CheckYourAnswersController @Inject()(
                                             override val messagesApi: MessagesApi,
                                             identify: IdentifierAction,
                                             getData: DataRetrievalAction,
                                             requireData: DataRequiredAction,
                                             val controllerComponents: MessagesControllerComponents,
-                                            view: CheckYourAnswersView
+                                            view: CheckYourAnswersView,
+                                            submissionService: SubmissionService
                                           ) extends FrontendBaseController with I18nSupport {
 
   def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
 
       val checkYourAnswersHelper = new CheckYourAnswersHelper(request.userAnswers)
 
-      val sections = Seq(AnswerSection(None, Seq()))
+      val sections = Seq(AnswerSection(None, Seq(
+        checkYourAnswersHelper.taxYearSelection,
+        checkYourAnswersHelper.whichSubscription,
+        checkYourAnswersHelper.subscriptionAmount,
+        checkYourAnswersHelper.employerContribution,
+        checkYourAnswersHelper.addAnotherSubscription,
+        checkYourAnswersHelper.yourEmployer,
+        checkYourAnswersHelper.yourAddress
+      ).flatten))
```

Comment:
Is it not worth doing this as a function in the helper e.g.

```
  def toList: Seq[AnswerRow] = {
    Seq(
      taxYearSelection,
      whichSubscription,
      subscriptionAmount,
      employerContribution,
      addAnotherSubscription,
      yourEmployer,
      yourAddress
    ).flatten
  }
```

Reply:
I would go with adding them here as it's clear what this controller is adding to this view, rather than moving to the helper. I understand this may need to change once we add more functionality.